pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/ServerCnx.java

```diff
@@ -332,6 +329,12 @@ protected void handleSubscribe(final CommandSubscribe subscribe) {
         final long consumerId = subscribe.getConsumerId();
         final SubType subType = subscribe.getSubType();
         final String consumerName = subscribe.getConsumerName();
+        final boolean isDurable = subscribe.getDurable();
```

Comment:
The flag is optional in the proto definition but it defaults to `true`, so there's no need to check for `hasDurable()`